### DISCUSSION TITLE
Laravel Echo connection established but not subscribed to channel(s)

### DISCUSSION THREAD

**Discussion by thannaske** [livewire/livewire#8417] [1. Help]
I'm currently trying to follow the documentation in regards of executing Livewire methods triggered by real-time notifications using Laravel Reverb and Laravel Echo. 

The main issue here seems to be that the Livewire Echo-implementation doesn't even try to subscribe to any channel whatsoever.

I followed the Livewire documentation and implemented the following code:

### Event
```php
class OrderDispatched implements ShouldBroadcast
{
    use Dispatchable, InteractsWithSockets, SerializesModels;

    public function __construct(public Order $order) {}

    public function broadcastOn(): array
    {
        return [
            new PrivateChannel('user.' . $this->order->user->id),
            new Channel('test'),
        ];
    }
}
```
As you can see, while testing I tried to dispatch the Event to a private **and** a public channel.

### Livewire Component
I tried both ways of listening to events, first by using the annotation ...

```php
#[On('echo:test, OrderDispatched')]
public function refresh()
{
    $this->dispatch('$refresh');
}
```

... and then by using the `getListeners()` method:

```php
public function getListeners()
{
    $user = auth()->user();

    return [
        "echo:test,OrderDispatched" => 'refresh',
        "echo-private:user.{$user->id}, OrderDispatched" => 'refresh',
    ];
}
```

### Issue
Now when reloading the page in the browser where the component is rendered, I cannot see any subscription event whatsoever, neither in Laravel Reverb nor in the browser inspector.

However, I can manually listen to the channel when using the JS console in the browser without any issue. What am I doing wrong?

**Comment by AdlyAlimin** [comment]
has you fix this? im currently facing this issue

**Reply by thannaske** [reply]
Unfortunately not 😕

**Reply by AdlyAlimin** [reply]
i have resolve it. just need to set broadcast name and can use in livewire. 

https://laravel.com/docs/11.x/broadcasting#broadcast-name
